multi_partner_learning.py

```diff
@@ -480,17 +524,33 @@ def collaborative_round_fit(model_to_fit, train_data, val_data, batch_size):
         """Fit the model with arguments passed as parameters and returns the history object"""
 
         x_train, y_train = train_data
-        history = model_to_fit.fit(
-            x_train,
-            y_train,
-            batch_size=batch_size,
-            epochs=1,
-            verbose=0,
-            validation_data=val_data,
-        )
-
+        if isinstance(model_to_fit, type(LogisticRegression())):
+            history = model_to_fit.fit(x_train, y_train)
+        else :
+            history = model_to_fit.fit(
+                x_train,
+                y_train,
+                batch_size=batch_size,
+                epochs=1,
+                verbose=0,
+                validation_data=val_data,
+            )
         return history
 
+    @staticmethod
+    def collaborative_round_evaluation(model_to_evaluate, evaluation_data):
+        """Evaluate the model with arguments passed as parameters and returns the history object"""
```

Comment:
Does it return the history for logistic regression ?

Reply:
Ah yes it mimics the history object, that is good !
We should just change the comments to say it returns` [loss, accuracy]`

Reply:
As we said, rename function to `evalute_model`